ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -75,7 +41,8 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
             onChange={e => {
               const newQuestionType = e.target.value as QuestionType
               setSelectedQuestionType(newQuestionType)
-              setQuestion(baseQuestions[newQuestionType])
+              // @ts-ignore
+              setQuestion({ ...baseQuestions[newQuestionType], ...question, type: newQuestionType })
```

Comment:
`ts-ignore`d because adding the `type` field here gives: 

`TS2322: Type '"checkbox" | "dropdown" | "medications" | "radiogroup" | "signaturepad" | "text"' is not assignable to type '"text"'.   Type '"checkbox"' is not assignable to type '"text"'.`

Not totally sure how to fix this.

Reply:
The only thing I can come up with is that this could be slightly improved to provide some type safety by casting the result instead of using `ts-ignore`.

```suggestion
              setQuestion({ ...baseQuestions[newQuestionType], ...question, type: newQuestionType } as Question)
```

Reply:
Good call, done in 64ffcd5a